src/components/EditServerDetails.js

```diff
@@ -26,6 +26,8 @@ class EditServerDetails extends Component {
   constructor(props) {
     super(props);
 
+    this.IS_INSTALLL = this.props.mode ? false : true;
```

Comment:
Can you fix this typo? INSTALL only has 2 L's

Reply:
ok will fix